sdcm/cluster_aws.py

```diff
@@ -873,10 +872,6 @@ def traffic_control(self, tcconfig_params=None):
             self.remoter.run('sudo bash -cxe "%s"' % tc_command)
 
     def install_traffic_control(self):
```

Comment:
seems `install_traffic_control` function is not needed any more

Reply:
it is been used by `nemesis.py`

Reply:
as yaron was saying, it is being using in nemesis.py.

and isn't related to this PR

Reply:
But now after this change the function is not installing anything... so the the is not exactly correct
and looking as the following:
```
    def install_traffic_control(self):
        return self.remoter.run("/sbin/tc -h", ignore_status=True).ok
```

Just checking and returns ok... 
and it was added only for amazon2...